### PR TITLE
commit ContactPageTest

### PR DESCRIPTION
**Pull request by DanaCatavei**

### CODE REVIEW

File: src/main/java/pages/ContactPage/ContactPage.java [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+        click(butonContact);
+    }
+    public void verifyEnableFrame() {
+        try {
+            waitPage().until(ExpectedConditions.visibilityOf(iframeLocator));
+            driver.switchTo().frame(iframeLocator);
+        } catch (Exception e) {
+            throw new NoSuchFrameException("No frame available!");
+        }
+        Assert.assertTrue(getPageTitle().equals("Suport ContApp"), PageLinksAndText.messageTestPageTitle.getValue());
```
Comment (ZahariaAlexC) [inline src/main/java/pages/ContactPage/ContactPage.java]: What Did you use this Assert , if you want to check to object?

File: src/main/java/pages/ContactPage/ContactPage.java [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+        try {
+            waitPage().until(ExpectedConditions.visibilityOf(fieldInputMessage));
+            click(fieldInputMessage);
+            addText(myText.replace("\\u000a", Keys.chord(Keys.SHIFT, Keys.ENTER)),fieldInputMessage);
+            click(buttonEmojiPicker);
+            click(valueEmojiHeart);
+            click(buttonEmojiPicker);
+            click(valueEmojiSmilingFace);
+            //click(buttonUploadFile);
+            addText("C:\\_DANA\\Screenshot 2023-08-01 124456.png",buttonUploadFile);
```
Comment (ZahariaAlexC) [inline src/main/java/pages/ContactPage/ContactPage.java]: hard coded this line : if someone wants to run your test, will failed--> your picture is not available in framework

File: src/main/java/pages/ContactPage/ContactPage.java [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+    public void inputMessage(String myText) throws IOException {
+        try {
+            waitPage().until(ExpectedConditions.visibilityOf(fieldInputMessage));
+            click(fieldInputMessage);
+            addText(myText.replace("\\u000a", Keys.chord(Keys.SHIFT, Keys.ENTER)),fieldInputMessage);
+            click(buttonEmojiPicker);
+            click(valueEmojiHeart);
+            click(buttonEmojiPicker);
+            click(valueEmojiSmilingFace);
+            //click(buttonUploadFile);
```
Comment (ZahariaAlexC) [inline src/main/java/pages/ContactPage/ContactPage.java]: remove it,  do not comment line in PR

**Review by ZahariaAlexC — CHANGES_REQUESTED** [review]

File: src/main/java/pages/ContactPage/ContactPage.java (hunk 1 above)
Comment (DanaCatavei) [reply src/main/java/pages/ContactPage/ContactPage.java]: Ok. Am elminat Assert

File: src/main/java/pages/ContactPage/ContactPage.java (hunk 2 above)
Comment (DanaCatavei) [reply src/main/java/pages/ContactPage/ContactPage.java]: Ok. I fixed!

**Review by ZahariaAlexC — APPROVED** [review]

**Review by teodorrdl — APPROVED** [review]

**Review by dicacristian — APPROVED** [review]

**Pull request merged**